src/main/Extensions/WindowsControlPanel/WindowsControlPanelItemsRepository.test.ts

```diff
@@ -0,0 +1,106 @@
+import { PowershellUtility } from "@Core/PowershellUtility/Contract";
```

Comment:
Please make sure that you always use `import type { ... } from ...` whenever possible.

```suggestion
import type { PowershellUtility } from "@Core/PowershellUtility/Contract";
```

Reply:
Done.